attachments.md

```diff
@@ -33,3 +33,8 @@ small_cover_land.jpg | Landscape          | 120
 
 ## Font files
 
+Font files MAY be added to a Matroska file as attachments so that the font file may be used to display an associated subtitle track. This allows the presentation of a Matroska file to be consistent in various environments where the needed fonts might not be available on the local system.
+
+To associate a font file with a subtitle track, the font file MUST be stored as an Attachment within the same Segment. The `TrackEntry Element` of the associated subtitle track SHOULD store an `AttachmentLink Element` that contains the value of the `FileUID Element` of the `AttachedFile Element` that stores the font file.
+
+For subtitle formats that reference a required font by name, such as SubStation Alpha (SSA/ASS), then the `AttachmentLink Element` is RECOMMENDED but not MANDATORY. If no `AttachmentLink Element` is used then the Matroska Reader SHOULD use the font(s) stored in Attachments that match the names of those needed by the subtitle encoding. If the subtitle encoding requires fonts that are not specified by an `AttachmentLink Element` or use names that do not match the `FileName` of any `AttachmentFile Elements` then the Matroska Reader SHOULD attempt to find a system font to use with the subtitle track.
```

Comment:
1. Superfluous "then" in "…/ASS), then the…".
2. Missing comma before "then" in "…is used, then the…"
3. Missing "s" here: "If the subtitle encoding …or uses names…" instead of "…or use names…"
4. Missing comma before "then" in "…`AttachmentFile Elements` then the…"
5. Maybe change the last sentence to state that the player should look up a system font with the name mentioned in the subtitle encoding?